Drop conflicting dprintf declaration.

ERESI does declare a prototype for dprintf, which conflicts with the
declaration in stdio.h, leading to the following compiler error:

../librevm/include/revm.h:604: error: conflicting types for ‘dprintf’
/usr/include/stdio.h:397: note: previous declaration of ‘dprintf’ was here

As the whole ERESI source code doesn't even mention dprintf in any
other place, dropping the declaration shouldn't cause any harm at all.

2010-04-07 Martin von Gagern

Index: eresi-0.82_beta2/librevm/include/revm.h
===================================================================
--- eresi-0.82_beta2.orig/librevm/include/revm.h
+++ eresi-0.82_beta2/librevm/include/revm.h
@@ -601,7 +601,6 @@ int		revm_isnbr(char *string);
 void		revm_workfiles_load();
 int		revm_implicit(revmcmd_t *actual);
 int	        revm_workfiles_unload();
-int		dprintf(int fd, char *format, ...);
 void	        revm_pht_print(elfsh_Phdr *phdr, uint16_t num, eresi_Addr base);
 char		*revm_fetch_sht_typedesc(elfsh_Word typenum);
 int             revm_sht_print(elfsh_Shdr *shdr, u_int num, char rtflag);
